suite/runsuite_wrapper.pl

```diff
@@ -391,6 +391,7 @@
                 'prof_pcs,thread_private|common.nativeexec_bindnow_opt' => 1, # i#2052
```

Comment:
Add a ref to #1807 in the Issue line since this is part of trying to get the post-merge suite green.

Reply:
Done.